.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# https://help.github.com/en/articles/about-code-owners
+# Default reviewers for everything
+* @spetrosi @nhosoi @richm
```

Comment:
you can omit spetrosi and nhosoi - but add yourself to this list

Reply:
ack, will change